docs/src/content/docs/api/calls.mdx

```diff
@@ -240,6 +245,11 @@ An optional query parameter called `priorCallId` can be provided to continue a p
         <td>string</td>
         <td>Unique identifier for the call.</td>
       </tr>
+      <tr>
+        <td class="font-mono">clientVersion</td>
+        <td>string</td>
+        <td>The version of the client that was used to join the call. Optionally can contain additional string defined during [`joinCall`](/sdk/#joincall)</td>
```

Comment:
Is it really necessary for us to update two places with this info?  Seems like we should either generate both places from our schema or change the docs to more of a "here are the nouns, here are the verbs" structure.

(Raising this both from a maintainability perspective and because it seems absurd to document clientVersion on the call creation response since it's always going to be null at this point.)

Reply:
i feel you. def something to do in the next big docs update. need a better wait to be DRY etc.